### ISSUE TITLE
Is it possible to mention my webui

### ISSUE THREAD

**Issue by yussufbiyik** [felixrosberg/FaceDancer#41]
Hello, 

I created a webui for this project for easier use and I was wondering if it was possible to mention it on the readme of this project, here's the link to it:
https://github.com/yussufbiyik/facedancer-webui

Thank you.

**Comment by adamgeddon1686** [comment]
I just downloaded this and will mess with it. I have never used gradio in code before but one thing I noticed right off the bat is how you go about entering into the script. I made a .bat file to enter into my version of facedancer. By doing so I can add its location to the path variable and call on it from whatever folder I choose. It also has the advantage of being able run the command to activate the environment before enter into the script and keeping it in one location if you ever need to change it. 

	@echo off

	call (path\to\conda)miniconda3\Scripts\activate.bat C:\Users\adamg\miniconda3\envs\(Env_NAME_Here)
	cd (path\to\FaceDancer)

	:loop
	python app.py
	set /p "repeat=Run FaceDancer again? (Y/N): "
	if /i "%repeat%"=="Y" goto :loop

**Comment by adamgeddon1686** [comment]
I just downloaded this and will mess with it. I have never used gradio in code before but one thing I noticed right off is how you go about entering into the script. I made a .bat file to enter into my version of facedancer. By doing so I can add its location to the path variable and call on it from whatever folder I choose. It also has the advantage of being able run the command to activate the environment before enter into the script and keeping it in one location if you ever need to change it. 

	@echo off

	call (path\to\conda)\Scripts\activate.bat (path\to\conda)\envs\(Env_NAME_Here)
	cd (path\to\FaceDancer)

	:loop
	python app.py
	set /p "repeat=Run FaceDancer again? (Y/N): "
	if /i "%repeat%"=="Y" goto :loop

Feel free to use this if you find it helpful.

**Comment by yussufbiyik** [comment]
Thank you, I was thinking about adding file selectors to manually add facedancer paths to the ui but this is also a viable solution too, i might add it. 

Also feel free to mess with it, there are probably many errors and things to be fixed so if you find any please let me know 😃

**Comment by adamgeddon1686** [comment]
oh something I noticed from using this approach is that you need to install gradio to your facedancer environment, not your main python environment. In your version you would do the opposite, ie install gradio to your python environment not your conda facedancer env

**Comment by yussufbiyik** [comment]
Yeah that's true, i never really used Python for this type of project and it was kinda like a hobby thing but i'm up to using that approach if it's better, would you recommend?

**Comment by adamgeddon1686** [comment]
I like it because then you can make a shortcut to your bat file and put it on your desktop. Or you can just type the name of the bat file in the file explorer bar if its on your path. I'm no expert though. I learned a lot of these things from ChatGPT and the rest of it from frustratingly combing stackoverflow. You can mess around with it if you would like.

**Comment by yussufbiyik** [comment]
Thank you, I am having a strange issue where ui doesn't update even though i update the code so I'll try to use an environment after fixing this issue

**Comment by felixrosberg** [comment]
Hi, I will try find time to test out your webui first. Thank you for involving yourself. If it works well I sure will endorse it!

**Comment by yussufbiyik** [comment]
Thank you, it's still in development and i am planning to improve it in time
